packages/@aws-cdk/aws-elasticloadbalancingv2-actions/lib/cognito-action.ts

```diff
@@ -0,0 +1,89 @@
+import * as cognito from '@aws-cdk/aws-cognito';
+import * as elbv2 from '@aws-cdk/aws-elasticloadbalancingv2';
+import { Duration } from '@aws-cdk/core';
+
+/**
+ * Properties for AuthenticateCognitoAction
+ */
+export interface AuthenticateCognitoActionProps {
+  /**
+   * What action to execute next
+   */
+  readonly next: elbv2.ListenerAction;
```

Comment:
Not optional? A following action is mandatory?

Reply:
Understood why this is later. Maybe add this - "Multiple actions form a linked chain; the chain must always terminate in a *(weighted)forward*, *fixedResponse* or *redirect* action." ?